contracts/scryptFramework.sol

```diff
@@ -62,6 +85,15 @@ contract ScryptFramework {
         state.inputHash = inputHash;
         return (false, state);
     }
+
+    /**
+      * @dev checks for equality of a and b's hash
+    *
+      * @param a the first equality operand
+      * @param b the second equality operand
+    *
+      * @return return true or false
```

Comment:
I'm really not a fan of these comments. If there is nothing to be gained from describing a parameter or a return value, I would prefer no description at all, because such description distract too much.

Reply:
point taken 👍